deepchem/models/torch_models/layers.py

```diff
@@ -6080,3 +6080,78 @@ def forward(self, dist: torch.Tensor, angle: torch.Tensor,
         output: torch.Tensor = (rbf[idx_kj].view(-1, n, k) *
                                 cbf.view(-1, n, 1)).view(-1, n * k)
         return output
+
+
+class HighwayLayer(torch.nn.Module):
+    """
+    Highway layer from "Training Very Deep Networks"
+    https://arxiv.org/abs/1507.06228
+
+
+    y = H(x)*T(x) + x*C(x), where
+
+    H(x): 1-layer neural network with non-linear activation
+    T(x): 1-layer neural network with sigmoid activation
+    C(X): 1 - T(X); As per the original paper
+
+    The output will be of the same dimension as the input
+
+     References
+    ----------
+    .. [1] Srivastava et al., "Training Very Deep Networks"
+
+    Examples
+    --------
+    >>> x = torch.randn(16, 20)
+    >>> highway_layer = torch_layers.HighwayLayer(d_input=x.shape[1])
+    >>> y = highway_layer(x)
+    >>> x.shape
+    torch.Size([16, 20])
+    >>> y.shape
+    torch.Size([16, 20])
+    """
+
+    def __init__(self,
+                 d_input: int,
+                 activation_fn: Union[Callable, str] = 'relu'):
+        """
+        Initializes the HighwayLayer.
+
+        Parameters
+            ----------
```

Comment:
Make sure to fix spacing

Reply:
Thanks for the review. I am not exactly sure where spacing needs to be fixed. I have made a commit removing some unnecessary new lines and added spacing between the operators. I hope you meant one of these, if not please let me know where spacing needs to be fixed.

Reply:
you will have to fix the spacing of the `----` below the parameters so that it lines up properly

Reply:
Thanks, I completely overlooked that one.